src/view/uniprotkb/components/ProteinOverviewView.tsx

```diff
@@ -18,6 +19,7 @@ export const ProteinOverview: FC<{
     _ => _.recommendedName.fullName.value
```

Comment:
I do realise you are following the `idx` examples/patterns, but two things comes to my mind when I see the `_` here:

1) In most FP languages, `_` is where the data is ignored. Basically a placeholder for something that we don't need to access.

2) A utility function from `Lodash` library.

Does it make sense to use a better parameter name here e.g. `o` or `obj`.

Reply:
Interesting point, I'll raise this as a broader question with you both (and @xwatkins) as throughout our code base (and in Franklin) we follow this pattern. Would prefer to fix - if that is the consensus - in a separate jira/issue and not as part of this PR.

Reply:
Updated.